packages/@jsii/java-runtime-test/project/src/test/java/software/amazon/jsii/testing/ComplianceTest.java

```diff
@@ -335,7 +335,7 @@ public void exceptions() {
         assertEquals(23, calc3.getValue());
         boolean thrown = false;
         try { calc3.add(10); }
-        catch (Exception e) { thrown = true; }
+        catch (RuntimeException e) { thrown = true; }
```

Comment:
What is this indentation style? Ewww 🤣

Reply:
that's been there since the first external release...yikes I'll update that